src/app/bs-navbar/bs-navbar.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'bs-navbar',
+  templateUrl: './bs-navbar.component.html',
+  styleUrls: ['./bs-navbar.component.scss']
```

Comment:
this file is empty so you should remove it and remove this line

Reply:
We end up using this function later on in Project tutorial. This is what it looks like now:

`import { Component, OnInit } from '@angular/core';
import { AuthService } from '../auth.service';

@component({
selector: 'bs-navbar',
templateUrl: './bs-navbar.component.html',
styleUrls: ['./bs-navbar.component.scss']
})
export class BsNavbarComponent {
constructor(public auth: AuthService) {}

logout() {
this.auth.logout();
}
}`